sdks/python/apache_beam/transforms/enrichment_handlers/feast_feature_store.py

```diff
@@ -95,13 +109,21 @@ def __init__(
       entity_id (str): entity name for the entity associated with the features.
         The `entity_id` is used to extract the entity value from the input row.
       feature_store_yaml_path (str): The path to a YAML configuration file for
-        the Feast feature store.
+        the Feast feature store. See
+        https://docs.feast.dev/reference/feature-repository/feature-store-yaml
+        for configuration options supported by Feast.
       feature_names: A list of feature names to be retrieved from the online
         Feast feature store.
       feature_service_name (str): The name of the feature service containing
         the features to fetch from the online Feast feature store.
       full_feature_names (bool): Whether to use full feature names
         (including namespaces, etc.). Defaults to False.
+      entity_row_fn: a lambda function that returns a dictionary with
```

Comment:
a few nits here:
1) Could you move `entity_id` down in the parameters list in this comment to match the ordering in the function definition?
2) Could you call out in `entity_row_fn`'s description that exactly one of `entity_id` and `entity_row_fn` must be defined?
3) Could you mention the input to this `entity_row_fn` (e.g. `a lambda function that takes in an incoming beam Row of data and returns a dictionary ...`

Reply:
sorry, missed this. done.